code/messaging/src/main/res/drawable/baseline_pending_20.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
What is this resource used for? `baseline_pending_20`...

Reply:
It was supposed to be a temporary one till I added shimmer for image loading. let me replace it

Reply:
Added shimmer effect